src/helpers/cauldron/position/getUserCollateralInfo.ts

```diff
@@ -0,0 +1,28 @@
+import type { BigNumber, Contract } from "ethers";
```

Comment:
In theory, this can all be replaced by the MarketLens, right? Should be inside the cauldronInfo call.